packages/components/src/DataList/components/DataListBulkActions/DataListBulkActions.test.tsx

```diff
@@ -0,0 +1,141 @@
+/* eslint-disable max-statements */
+import React, { PropsWithChildren } from "react";
+import { render, screen, within } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import {
+  DataListContext,
+  defaultValues,
+} from "@jobber/components/DataList/context/DataListContext";
+import { DataListBulkActions } from ".";
+import { DataListAction } from "../DataListAction";
+import { DataListHeaderCheckbox } from "../DataListHeader/DataListHeaderCheckbox";
+import { BREAKPOINT_SIZES, Breakpoints } from "../../DataList.const";
+
+const handleEditClick = jest.fn();
+const mockBulkActionsComponent = jest.fn().mockReturnValue(
+  <DataListBulkActions>
+    <DataListAction icon="edit" label="Edit" onClick={handleEditClick} />
+    <DataListAction icon="email" label="Email" />
+    <DataListAction icon="note" label="Note" />
+    <DataListAction label="Delete" />
+  </DataListBulkActions>,
+);
+
+const mockHeader = { label: "Name" };
+const mockData = [{ id: 1, label: "Luke Skywalker" }];
+
+Object.defineProperty(window, "matchMedia", {
+  writable: true,
+  value: jest.fn().mockImplementation(query => ({
+    matches: true,
+    media: query,
+    onchange: null,
+    addListener: jest.fn(), // deprecated
+    removeListener: jest.fn(), // deprecated
+    addEventListener: jest.fn(),
+    removeEventListener: jest.fn(),
+    dispatchEvent: jest.fn(),
+  })),
+});
+
+const originalMatchMedia = window.matchMedia;
+
+afterAll(() => {
+  window.matchMedia = originalMatchMedia;
+});
+
+describe("DataListBulkActions", () => {
+  it("should render the 4 buttons", () => {
+    renderComponent();
+    const overflowContainer = screen.getByTestId(
+      "ATL-DataListFilters-Container",
+    );
+
+    expect(within(overflowContainer).getAllByRole("button")).toHaveLength(4);
+  });
+
+  it("should render a more action", () => {
+    renderComponent();
+
+    expect(screen.getByLabelText("More actions")).toBeInTheDocument();
+  });
+
+  it("should call the onClick handler when clicking the edit button", () => {
+    renderComponent();
+    userEvent.click(screen.getByLabelText("Edit"));
+    expect(handleEditClick).toHaveBeenCalledTimes(1);
```

Comment:
should make sure it actually doesn't call it with a param. other than that, looks pretty good!

```suggestion
    expect(handleEditClick).toHaveBeenCalledTimes(1);
    expect(handleEditClick).toHaveBeenCalledWith();
```